### PR TITLE
Changes and additions for firewalled-replit.com

### PR DESCRIPTION
**Pull request by qirtaiba**
This adds a new docs section on Firewalled Replit.

### CODE REVIEW

File: docs/firewalled-replit/firewalled-replit-faq.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+sidebar_position: 2
+---
+
+# Firewalled Replit FAQ
+
+_Read more about this change at [our blog post](https://blog.replit.com/computing-superpower-at-school). Also, see our [commitment to user safety.](https://docs.google.com/document/d/1rGjlPhRIFDYVkLkP_nO0Db6o1_9EmGvSnLt9cHacE5M/edit)_
```
Comment (Bardia95) [inline docs/firewalled-replit/firewalled-replit-faq.md:7]: can we change this link from a google doc to a markdown repl or a doc hosted on the docs site

**Review by Bardia95 — CHANGES_REQUESTED** [review]

File: docs/firewalled-replit/firewalled-replit-faq.md (hunk 1 above)
Comment (Bardia95) [reply docs/firewalled-replit/firewalled-replit-faq.md]: thanks!

**Review by qirtaiba — COMMENTED** [review]
Fixed URL that pointed to a Google Doc

**Comment by qirtaiba** [comment]
Please merge by May 31 @Bardia95. Thanks!

**Comment by Bardia95** [comment]
thanks! can merge now if you're ready

**Pull request merged**